docs/index.md

```diff
@@ -30,7 +30,7 @@ Core utilities such as Tracer, Logger, Metrics, and Event Handler will be availa
 
 Powertools is available in the following formats:
 
-* **Lambda Layer**: [**arn:aws:lambda:{region}:017000801446:layer:AWSLambdaPowertoolsPython:19**](#){: .copyMe}:clipboard:
+* **Lambda Layer**: [**arn:aws:lambda:{region}:264025721901:layer:AWSLambdaPowertoolsNodeJS:1**](#){: .copyMe}:clipboard:
```

Comment:
I know the runtime is NodeJS, but I'm a bit on the fence with calling the Layer something different than the project's name (TypeScript). 

What do you think?

Reply:
I was calling it TypeScript and then I realized that if first if we tight it to the runtime then it should be NodeJS second it is as well for JavaScript ... but ok to revert to TypeScript

Reply:
Let's talk about it and hear more opinions before making a decision. I'm still not 100% sure which should be